Makefile

```diff
@@ -20,5 +20,28 @@ $(call add-bindata,installer,./pkg/operator/staticpod/controller/installer/manif
 $(call add-bindata,staticpod,./pkg/operator/staticpod/controller/prune/manifests/...,bindata,bindata,./pkg/operator/staticpod/controller/prune/bindata/bindata.go)
 $(call add-bindata,auditpolicies,./pkg/operator/apiserver/audit/manifests/...,bindata,bindata,./pkg/operator/apiserver/audit/bindata/bindata.go)
 
+update-bindata: update-bindata-podnetworkconnectivitycheck
```

Comment:
we have build machinery for that cc @tnozicka

Reply:
@sttts , @tnozicka I had to go on my own here in order to specify the `-ignore` parameter.

Reply:
I've opened https://github.com/openshift/build-machinery-go/pull/25 to add the ability for the future.

Reply:
We need ignore because the CRD manifests are in the types directory?

We could also just copy the manifest into some directory of the image and open it from there.

Reply:
Change to copy and gen.